src/mympi.cpp

```diff
@@ -259,10 +287,10 @@ int max_to_all(int in) {
 }
 
 ivec max_to_all(const ivec &pt) {
-  int in[5], out[5];
+  long int  in[5], out[5];
```

Comment:
Seems like there should be a check somewhere that `sizeof(long int) == sizeof(meep::integer)`.